roxie/ccd/ccdmain.cpp

```diff
@@ -715,6 +715,8 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         traceLevel = topology->getPropInt("@traceLevel", runOnce ? 0 : 1);
         if (traceLevel > MAXTRACELEVEL)
             traceLevel = MAXTRACELEVEL;
+        if (traceLevel && topology->hasProp("logging/@disabled"))
+            topology->setPropBool("logging/@disabled", false);
```

Comment:
If a config file had disabled=true, I'd have thought that would override other logging settings like traveLevel.
But this means if traveLevel>0, it ignores @disabled flag.

Is this logic required?

Reply:
It's so that I can set traceLevel=1 when running a standalone program (where default logging is disabled)